package.json

```diff
@@ -5,10 +5,10 @@
     "build-js": "webpack",
     "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle",
     "format-python": "black --line-length 79 tests webapp",
-    "lint-scss": "sass-lint static/**/*.scss --verbose --no-exit",
+    "lint-scss": "stylelint static/**/*.scss",
     "python-lint": "flake8 webapp tests && black --check --line-length 79 tests webapp",
     "serve": "./entrypoint 0.0.0.0:${PORT}",
-    "test": "npm run python-lint && npm run test-python && sass-lint static/**/*.scss --verbose --no-exit",
+    "test": "npm run python-lint && npm run test-python && stylelint static/**/*.scss",
```

Comment:
Can you switch this to `npm run lint-scss` instead of calling styleliint directly as the commands could get out of sync